utest/tidy/test_argument_validation.py

```diff
@@ -17,9 +17,14 @@ def test_valid_space_count(self):
         assert_equal(opts['spacecount'], 42)
 
     def test_invalid_space_count(self):
-        error = '--spacecount must be an integer greater than 1.'
+        error = '--spacecount must be of value "preserve" or an integer greater than 1.'
```

Comment:
Howa about : `--spacecount must be either "preserve" or an integer greater than 1.'`?